_layouts/default.html

```diff
@@ -40,6 +40,8 @@ <h1 class="header">{{ site.data.package.name }}</h1>
           {% elsif site.data.package.github %}
           <li><a class="buttons github" href="{{ site.data.package.github }}">View On GitHub</a></li>
           {% endif %}
+          <li><a class="clean" href="{{ site.data.package.doc-html }}">Manual</a></li>
```

Comment:
That's not a button, though, it's a navigation link (nav link), no? Not meant to nitpick, I just was confused by the PR title ("button ?!?"), but the change itself makes sense.